model.py

```diff
@@ -1889,8 +1889,8 @@ def author_contributor_tiers(cls):
         yield cls.PERFORMER_ROLES
 
     @classmethod
-    def lookup(cls, _db, name=None, viaf=None, lc=None, aliases=None,
-               extra=None, create_new=True):
+    def lookup(cls, _db, sort_name=None, viaf=None, lc=None, aliases=None,
+               extra=None, create_new=True, name=None):
```

Comment:
Same here -- remove `name` altogether while getting the other components to work.